machine_learning/lstm/lstm_prediction.py

```diff
@@ -0,0 +1,66 @@
+import pandas as pd
+import numpy as np
+from sklearn.preprocessing import MinMaxScaler
+from keras.models import Sequential
+from keras.layers import LSTM,Dense
+
+def clean_data(data_file):
+	'''
+	This function will read and clean data
+	input: CSV file containing data
+	output: Data which will be useful for training
+	'''
+	df = pd.read_csv(data_file, header=None)
+	print(df.head())
+	len_data = df.shape[:1][0]
+	actual_data = df.iloc[:,1:2]
+	actual_data = actual_data.values.reshape(len_data,1)
+	scl = MinMaxScaler()
+	actual_data = scl.fit_transform(actual_data)
+	look_back = 10
+	forward_days = 5
+	periods = 20
+	division = len_data - periods*look_back
+	train_data = actual_data[:division]
+	test_data = actual_data[division-look_back:]
+	print(train_data,test_data)
+	train_x,train_y=[],[]
+	test_x,test_y=[],[]
+	for i in range(0,len(train_data) - forward_days - look_back + 1,1):
```

Comment:
Why start with zero and end with one.  These are not needed and only distract your reader.  Same a few lines below.